src/hott/06-contractible.rzk.md

```diff
@@ -110,69 +110,71 @@ A type is contractible if and only if its terminal map is an equivalence.
 
 ```rzk
 #def terminal-map-is-equiv
-  (A : U)
+  ( A : U)
   : U
   := is-equiv A Unit (terminal-map A)
 
 #def contr-implies-terminal-map-is-equiv-retr
-  (A : U)
-  (AisContr : is-contr A)
+  ( A : U)
+  ( A-is-contr : is-contr A)
```

Comment:
@fredrik-bakke I asked elsewhere about A-is-contr vs is-contr-A. Why do you prefer the latter?

I'm inclined towards "A-is-contr" because it reads correctly when you are reading the statement or proof of a theorem. But I'm open to being persuaded.

Reply:
Hi! Please see my answer in that original thread: https://github.com/emilyriehl/yoneda/pull/21#issuecomment-1628556942